Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE_README.md

```diff
@@ -8,25 +8,33 @@ On Dec. 14 2021, another vulnerability was discovered related the log4j 0-day ex
 
 Apache Log4j 2.x <= 2.15.0-rc1
 
-This playbook should be triggered manually.
-Please create a new incident and choose the **CVE-2021-44228 - Log4j RCE** playbook and **Rapid Breach Response** incident type.
-
 **The playbook includes the following tasks:**
 
-* Collect related known indicators from several sources.
-* Indicators and exploitation patterns hunting using PAN-OS, Cortex XDR and SIEM products.
-* Block indicators automatically or manually.
-
-**Mitigations:**
-* Apache official CVE-2021-44228 patch.
-* Unit42 recommended mitigations.
-* Detection Rules.
-    * Snort
-    * Suricata
-    * Sigma
-    * Yara
-
-More information:
+* Collecting Indicators - related known indicators from several sources.
+* Investigation and Hunting - indicators and exploitation patterns hunting using PAN-OS, Cortex XDR and SIEM products.
+* Remediation - Blocking indicators automatically or manually base on **playbook inputs**.
+* Mitigations:
+    * Apache official CVE-2021-44228, CVE-2021-45046 patch.
+    * Unit42 recommended mitigations.
+    * Detection rules for.
+        * Snort
+        * Suricata
+        * Sigma
+        * Yara
+
+**How to Use The Playbook**
+1. Create a new incident with the incident type "Rapid Breach Response" and choose the "CVE-2021-44228 - Log4j RCE" playbook.
+    1. The playbook can be configured as a job as newer indicators might be pulled in. Just create a new job with the incident type and playbook mentioned above.
+2. Look and change the playbook inputs. For example, 
+    1. To run the shell scripts on XDR endpoints - you will need to change "XDRScriptExecution" and the "XDREndpointIDs" playbook inputs. Note that running the scripts on **all** XDR connected linux endpoints might impact performance on the XDR (depending on number of endpoints).
+    2. Change the SIEM search time frame as requested (default is 1 day). Is you are using Splunk , you can change the index by changing the "SplunkSourcetype" playbook input.
```

Comment:
Should be **If not Is** you are using Splunk , you can change the index by changing the "SplunkSourcetype" playbook input.